js/security-headers.js

```diff
@@ -24,7 +24,8 @@ export default {
     connectSrc: [
       `'self'`,
       url.parse(env.PULSE_API).host ||
-        `https://network-pulse-api-staging.herokuapp.com/`
+        `https://network-pulse-api-staging.herokuapp.com/`,
+      `https://www.mozilla.org/en-US/newsletter/`
```

Comment:
```suggestion
        `https://www.mozilla.org/en-US/newsletter/`
```

Reply:
Indentation of this line is fine. The 2nd item in the array (```      url.parse(env.PULSE_API).host ||`https://network-pulse-api-staging.herokuapp.com/```)is long + the Prettier's formatting is making indentation of this array look kinda odd 😂